### PR TITLE
Fix white background once signed into elearning

### PR DESCRIPTION
**Pull request by prstorero**
Since the developers may not have access to the eLearning portal, I have provided screenshots that demonstrate the fix.

image

image

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -5,6 +5,15 @@ svg
 
 ================================
 
+elearning.utdallas.edu/webapps/portal/execute/tabs/tabAction?tab_tab_group_id=_1_1
```
Comment (alexanderby) [inline src/config/dynamic-theme-fixes.config]: Let's shorten the URL

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -5,6 +5,15 @@ svg
...
+NO
```
Comment (alexanderby) [inline src/config/dynamic-theme-fixes.config]: We have `INVERT` and `NO INVERT` commands, `NO` does nothing

Comment (prstorero) [reply src/config/dynamic-theme-fixes.config]: That's what I kept trying, but every time I would hit apply, the developer tool would move the NO up there for some reason. I'll move it back down like it's supposed to be.

**Comment by prstorero** [comment]
I tried editing the original commit, but I guess it made a new one with the requested changes. This is my first pull request, so I'm unfamiliar with how to make changes once requested.

**Pull request merged**